thirdparty/README.md

```diff
@@ -62,8 +62,8 @@ Use UI font if exists, because it has tight vertical metrics and good for UI.
 ### Hack Regular
 
 - Upstream: https://github.com/source-foundry/Hack
-- Version: 2.020
-- License: Hack Open Font License v2.0
+- Version: 3.000
+- License: MIT + Bitstream Vera License
```

Comment:
We're using it under the terms of the MIT, so we just need to name this one IMO.

Reply:
From what I understand of Hack's licensing, it's licensed under MIT *plus* Bitstream Vera restrictions (since Hack is based on DejaVu Sans Mono, which is in turn based on a Bitstream Vera font); it is not dual-licensing.

Reply:
Hm, you're right.